.github/scripts/process-plugins.py

```diff
@@ -40,21 +40,29 @@
     packaging_file = open(filepath)
     packaging = json.load(packaging_file)
     packaging_file.close()
-    packaging_path = re.search('.*\/(centreon-plugin-.*)\/pkg.json', filepath).group(1)
-
-    if not packaging_path == packaging["pkg_name"]:
-        packaging_path = packaging["pkg_name"]
+    packaging_path = packaging["pkg_name"]
 
     directory_path = re.search('^(.+)\/pkg.json', filepath).group(1)
 
     if common:
+        # if the common flag is true, then all packages are included
         list_packages.add(packaging_path)
     elif directory_path in packages:
+        # if a package file is changed or created, then the package is included
         list_packages.add(packaging_path)
     else:
+        # is a source code file is changed or created
```

Comment:
I think the comments after this are hard to understand.
else: 
# if the file changed is neither from common or a packaging file, check wich package include it.
# then all the included files... -> #  for each package file, check if the file dirname is present in the package file.